### PR TITLE
Fix `conflict-marker` spacing

### LINKED ISSUE #4117: `conflict-marker` icon is different from the screenshot.
conflict-marker is the motivation I installed refined-github. Thank you.
But the '!' icon is different from what I expected.(it looks the fill is inverted)
스크린샷 2021-03-17 오후 12 02 34
I don't know it is bug or my browser.(I don't apply any other setting or ui plugin)

MacOS 10.15.7, Chrome 88

### PR DESCRIPTION
**Pull request by cheap-glitch**
Fixes #4117 

## Test URLs

https://github.com/eslint/eslint/pulls?q=is%3Apr+is%3Aopen+sort%3Aupdated-desc+comments+

## Screenshot

With `align-issue-labels` enabled (no change):
<table>
<tr>
	<th>Before
	<td>

image
<tr>
	<th>After
	<td>

image
</table>

With `align-issue-labels` **disabled**:
<table>
<tr>
	<th>Before
	<td>

image
<tr>
	<th>After
	<td>

image
</table>

### CODE REVIEW

File: source/features/conflict-marker.css [hunk 1]
```diff
@@ -2,3 +2,8 @@
 	vertical-align: middle;
 	color: var(--color-icon-tertiary, #ccc);
 }
+
+/* Add some padding after the issue title for when align-issue-labels is disabled #4117 */
```
Comment (yakov116) [inline source/features/conflict-marker.css:6]:
```suggestion
/* Padding needed if `align-issue-labels` is disabled #4117 */
```

Not sure

**Comment by fregante** [comment]
I didn't notice you had already opened a PR 😅 check out my comment in the original issue, this probably isn't the right solution

**Review by cheap-glitch — COMMENTED** [review]
Both `align-issue-labels` and `conflict-marker` enabled:

image

Just `align-issue-labels`:

image

Just `conflict-marker`:

image

File: source/features/align-issue-labels.css [hunk 2]
```diff
@@ -8,9 +8,8 @@
 	order: 1;
 }
 
-.rgh-align-issue-labels .js-issue-row .h4, /* Title */
 .rgh-align-issue-labels .js-issue-row .d-inline-block.mr-1 { /* Build status */
-	padding-right: 4px; /* Space before the build status */
+	padding-left: 4px;
```
[the remark this answers is not in the document]
Comment (cheap-glitch) [reply source/features/align-issue-labels.css:12]: This is a pure CSS feature though, is it worth it to add JS just for this?

[the remark this answers is not in the document]
Comment (fregante) [reply source/features/align-issue-labels.css]: Ah indeed I thought that the spacing here was exactly due to the other feature changed in this PR, but instead it's a native icon.

**Comment by fregante** [comment]
I changed the title because the fix is mostly about doing things right rather than having code that depends on each other. If one day GitHub drops `align-issue-labels` then the alignment also breaks.

This has the welcome side effect that it helps people who disable features

**Pull request merged**